scripts/jenkins/ConfigureUbuntu1404VagrantEnvironment.sh

```diff
@@ -40,7 +40,7 @@ sed -i s/"QMAKE_CXX=g++"/"#QMAKE_CXX=g++"/g LocalConfig.pri
 sed -i s/"#QMAKE_CXX=ccache g++"/"QMAKE_CXX=ccache g++"/g LocalConfig.pri   
 
 # Make sure we are not running
-vagrant halt
+vagrant halt -f
```

Comment:
Does this need to be in other places like the CentOS vagrant scripts?  Not sure just something to think about.

Reply:
This was an experiment that had no effect. I'll remove it. I think there are larger issues at play, with respect to using the AWS vagrant boxes.